### PR TITLE
Adding frontend initial flow with material ui.

### PR DESCRIPTION
**Pull request by 6rivu**
Added home, mint-profile page, social graph page.

### CODE REVIEW

**Review by ewerx — APPROVED** [review]
I have no code comments. I don't know if you want design comments yet or we should save this for late, but the fonts and styling could use a pass.

File: src/components/buttons/connectwallet.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Button } from "@mui/material";
+import { Link } from "react-router-dom";
+
+require('react-dom');
+window.React2 = require('react');
+console.log(window.React1 === window.React2);
```
Comment (gabsong) [inline src/components/buttons/connectwallet.js:7]: Does this need to be included for the deploy?

File: src/components/buttons/mint.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Button } from "@mui/material";
+
+require('react-dom');
+window.React2 = require('react');
```
Comment (gabsong) [inline src/components/buttons/mint.js:5]: Could you leave a comment explaining the need of doing this?

File: src/components/socialgraph.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Grid } from "@mui/material";
+
+require('react-dom');
```
Comment (gabsong) [inline src/components/socialgraph.js:4]: nit: why are you using cjs require as opposed to ES6 import?

File: src/components/buttons/connectwallet.js (hunk 1 above)
Comment (6rivu) [reply src/components/buttons/connectwallet.js:7]: No, actually I included this for debugging the code incase two different versions of react is installed by mistake. I faced this issue while running the frontend on my local server. You can follow the issue here - [https://github.com/facebook/react/issues/2402](https://github.com/facebook/react/issues/2402)

File: src/components/buttons/mint.js (hunk 2 above)
Comment (6rivu) [reply src/components/buttons/mint.js:5]: Checking if two different versions of react have been installed for the app.js and the components. The same issue that I've linked to above.

File: src/components/socialgraph.js (hunk 3 above)
Comment (6rivu) [reply src/components/socialgraph.js:4]: Just copy-pasted the debugging code from stackoverflow :p to solve the issue.

**Pull request merged**